pymilvus/client/entity_helper.py

```diff
@@ -70,6 +70,43 @@ def entity_to_json_arr(entity: Dict):
     return convert_to_json_arr(entity.get("values", []))
 
 
+def convert_to_array_arr(objs: List[Any]):
+    arr = []
+    for obj in objs:
+        arr.append(convert_to_array_arr(obj))
+    return arr
+
+
+def convert_to_array(obj: List[Any], field_info: Any):
+    field_data = schema_types.ScalarField()
+    element_type = field_info.get("element_type", None)
+    if element_type == DataType.BOOL:
+        for value in obj:
+            field_data.bool_data.data.append(value)
```

Comment:
Please use extend to simplify this, and return as soon as possible to reduce elif
```python
field_data.bool_data.data.extend(obj)
return
```